app/components/currency-amount.js

```diff
@@ -0,0 +1,44 @@
+import classic from 'ember-classic-decorator';
+import { computed } from '@ember/object';
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+const locales = {
+  'de'      : 'de-DE',
+  'zh_Hans' : 'zh-CN',
+  'zh_Hant' : 'zh-TW'
+};
+
+const currencyCode = {
+  'bn'      : 'INR',
+  'de'      : 'DEM',
+  'en'      : 'USD',
+  'es'      : 'EUR',
+  'fr'      : 'EUR',
+  'hi'      : 'INR',
+  'id'      : 'IDR',
+  'ja'      : 'JPY',
+  'pl'      : 'PLN',
+  'ru'      : 'RUB',
+  'th'      : 'THB',
+  'vi'      : 'VND',
+  'zh_Hans' : 'CNY',
+  'zh_Hant' : 'CNY',
+  'ko'      : 'KPW'
+};
```

Comment:
Completely unnecessary. Currency code is already being passed in the component. How sound is that logic that would you display USD 2000 as INR 2000 if the user has Hindi selected? Bizzare